staff/oriol-bover/cv/index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Oriol Bover CV</title>
+</head>
+<body>
+    <!-- Header section -->
+    <header>
+        <h1>CURRICULUM VITAE</h1>
+    </header>
+    <!-- End header section -->
+    <!-- Basic info section-->
+    <section>
+        <h2>Basic information</h2>
+        <img src="#" alt="Oriol Bover profile image">
+        <ul>
+            <li>Name: Oriol Bover Vila</li>
+            <li><address>Canigo 41 Vic (Barcelona)</address></li>
```

Comment:
You can use <adress> tag with all of your contact info, as w3schools state: [source](https://www.w3schools.com/tags/tag_address.asp)